### ISSUE TITLE
FR: Support LInuxArm(64) target?

### ISSUE THREAD

**Issue by teras** [kgit2/kommand#12]
Hello.
Is there any reason why linux/arm is not supported? In theory the native API should be the same and no difference should be present.

I tried to hack the build script myself, but my still inexperience with kotlin/native, plus the added complexity of publishing options made the project uncompilable on my system.

**Comment by BppleMan** [comment]
You are really lucky!

`kommand` dependent on `ktor-io`, last month and earlier, that is before ktor 2.3.3, ktor-io of the linuxArm64 platform was not supported.

Also I don't have a suitable linuxArm64 test device, so I can't guarantee all test cases.

But now that ktor has released 2.3.4, I think it is possible to add linuxArm64 support to `kommand`, but it will take some time.

At the same time, I will also re-optimize build.gradle.kts, but if you have no publish relevant account information, you can also build kommand locally.

**Comment by teras** [comment]
No worries, there's no hurry. Right now I am still learning K/N and launching applications _the easy way_ was a real miss for me, and it's not as simple as other C functions. Finding your project was a great relief and an invest for the future.

Being on this position, although again right now I don't have ARM needs, I do have them in general, as an IoT fan.

Keep up the good work... your library is important. Please let's not close the issue yet, as a reminder of things to come. Thanks again

**Comment by teras** [comment]
"Fixing" the gradle script for me was easy.
I just had to override a couple of variables for publishing

```kotlin
val ossrhUsername: String = ""
val ossrhPassword: String = ""
```

**Comment by BppleMan** [comment]
> 对我来说“修复”gradle 脚本很容易。 我只需要重写几个变量即可发布
> 
> ```kotlin
> val ossrhUsername: String = ""
> val ossrhPassword: String = ""
> ```

This is correct, as long as the publish task is not executed

But I have released 1.1.0
linuxArm64 is officially supported and build.gradle.kts fixed
